fastapi_cache/backends/dynamodb.py

```diff
@@ -46,58 +46,63 @@ async def init(self) -> None:
         ).__aenter__()
 
     async def close(self) -> None:
-        self.client = await self.client.__aexit__(None, None, None)
+        if self.client:
+            await self.client.__aexit__(None, None, None)
+            self.client = None
 
     async def get_with_ttl(self, key: str) -> Tuple[int, Optional[bytes]]:
-        response = await self.client.get_item(TableName=self.table_name, Key={"key": {"S": key}})
+        if self.client:
```

Comment:
Better to have
```
if not self.client:
    return 0, None
```
to avoid deeply nested ifs and changing the indention of everthing below.